src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -456,7 +457,7 @@ def getClusterAd(self):
         ad['Cmd'] = self.scriptFile
         ad['Out'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".out")')
         ad['Err'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".err")')
-        ad['UserLog'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".log")')
+        ad['UserLog'] = classad.ExprTree('strcat(Iwd, "/condor.", ClusterId, ".", ProcId, ".log")')
```

Comment:
Not really a problem, but our `condorq` alias prints the `UserLog` ad in its output. Without the absolute path, we cannot easily see the condor logs location for that job.

I'll just set the absolute path to all of them then.